#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7648317683:AAFmAO79DQAmv5FxaO1L0Y0i31zUpVUGSvU")
    API_ID = int(os.environ["API_ID", 27498866]
    API_HASH = os.environ["API_HASH", "96fbb6ad2e11ab04e83ca09ef3f42455"]
    AUTH_USERS = "6488911325"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
